### PR TITLE
Docs and http timeout fixes

### PR DESCRIPTION
**Pull request by ljader**
Hi.
I've fixed some docs and added setting of all 3 of HttpClient timeouts.
(overall I consider this a "small change")

Thanks in advance for reviewing this.

### CODE REVIEW

File: src/main/java/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStep.java [hunk 1]
```diff
@@ -100,7 +100,7 @@ public String getFunctionName() {
         @Override
         @Nonnull
         public String getDisplayName() {
-            return "office365ConnectorSend";
+            return "Send build notifications to Microsoft Teams or Outlook";
```
Comment (damianszczepanik) [inline src/main/java/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStep.java]: how bout "to Office365" as I believe that MS Teams and Outlook are not the only one which accepts hooks?

Comment (damianszczepanik) [reply src/main/java/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStep.java]: also it does not have to be 'build' notification - it can be any notification :)

File: src/main/resources/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStep/help.html [hunk 2]
```diff
@@ -1,9 +1,3 @@
-<div>
-  This is the main text in a message attachment, and can contain standard message markup.
-  The content will automatically collapse if it contains 700+ characters or 5+ linebreaks, and will display a "Show
-  more..." link to expand the content.
-  Notification may include global variables, for example environment and currentBuild variables:<br>
-  <code>
-    office365ConnectorSend message:"started ${env.JOB_NAME} ${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)"
-  </code>
+<div align="help">
+  Step for sending build notifications to Microsoft Teams or Outlook.
```
Comment (damianszczepanik) [inline src/main/resources/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStep/help.html]: by this we lost some example which might be helpful for newbies

File: src/test/java/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStepTest.java [hunk 3]
```diff
@@ -207,7 +207,7 @@ public void getDisplayName_ReturnsFunctionName() {
         String displayName = descriptor.getDisplayName();
 
         // then
-        assertThat(displayName).isEqualTo("office365ConnectorSend");
+        assertThat(displayName).isNotEqualTo("office365ConnectorSend");
```
Comment (damianszczepanik) [inline src/test/java/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStepTest.java]: poor assertion

File: src/main/java/jenkins/plugins/office365connector/HttpWorker.java [hunk 4]
```diff
@@ -116,6 +115,9 @@ private HttpClient getHttpClient() {
                 }
             }
         }
+        client.getParams().setConnectionManagerTimeout(timeout);
```
Comment (damianszczepanik) [inline src/main/java/jenkins/plugins/office365connector/HttpWorker.java]: what is the motivation for this change?

Comment (ljader) [reply src/main/java/jenkins/plugins/office365connector/HttpWorker.java]: so that method "getHttpClient()" would hide all httpclient creation logic - and the addition of next 2 timeouts wouldn't clutter the "run" method

File: src/main/java/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStep.java (hunk 1 above)
Comment (ljader) [reply src/main/java/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStep.java]: Thx, I've pushed a reword - please check.

File: src/test/java/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStepTest.java (hunk 3 above)
Comment (ljader) [reply src/test/java/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStepTest.java]: Thx, I've changed a little bit this test case - please check.

File: src/main/resources/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStep/help.html (hunk 2 above)
Comment (ljader) [reply src/main/resources/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStep/help.html]: all of the deleted text is also available in help file for "message" field - so it's not lost.
If you want, I could split this text into those 2 files (help.html and help-message.html)
so the help.html would include the general information, and the help-message.html would describe only the "message" part.
